test/functional/api/cas/cli.py

```diff
@@ -215,13 +215,14 @@ def set_param_cleaning_alru_cmd(cache_id: str, wake_up: str, staleness_time: str
                                 shortcut: bool = False):
     add_param = ""
     if wake_up is not None:
-        add_param += (" -w " if shortcut else " --wake-up ") + wake_up
+        add_param += (" -w " if shortcut else " --wake-up ") + str(wake_up)
```

Comment:
Looks like you've change this function to accept non-string values of "wake_up" and other parameters. If that's your intention, please remove "str" type specification from argument list.

Reply:
True that! Removed type hints here